zerver/models.py

```diff
@@ -627,16 +627,13 @@ def get_realm_emoji_dicts(realm: Realm,
     for realm_emoji in query.all():
         author = None
```

Comment:
I think we should change this to `author_id = None`. We can then have `author_id=author_id` on line 636. 'author_id` would then be `none` when `realm_emoji.author` is false.

Reply:
When I passed author_id to tests it shows me error, can you help me with ```test_emoji_realm.py``` modifications too?

Reply:
You'll basically get errors at places where `test_realm_emoji.py` checks for the `author` dict. Modifying the code to check `author_id` instead should work.

Reply:
You should first make the changes to `models.py` only and then run `test-backend` to see where you get errors.